README.md

```diff
@@ -12,69 +12,41 @@ issuance and validation.
 _passport-azure-ad_ has been tested to work with both [Microsoft Azure Active Directory](https://azure.microsoft.com/en-us/services/active-directory/) 
 and with [Microsoft Active Directory Federation Services](http://en.wikipedia.org/wiki/Active_Directory_Federation_Services).
 
-## Security Vulnerability in Versions < 1.4.6 and 2.0.0
+## 1. Security Vulnerability in Versions < 1.4.6 and 2.0.0
 _passport-azure-ad_ has a known security vulnerability affecting versions <1.4.6 and 2.0.0. Please update to >=1.4.6 or >=2.0.1 immediately. For more details, see the [security notice](https://github.com/AzureAD/passport-azure-ad/blob/master/SECURITY-NOTICE.MD).
 
-## Versions
-Current version - 3.0.0  
+## 2. Versions
+Current version - 3.0.1  
 Minimum  recommended version - 1.4.6  
 You can find the changes for each version in the [change log](https://github.com/AzureAD/passport-azure-ad/blob/master/CHANGELOG.md).
 
-## Contribution History
+## 3. Contribution History
 
 [![Stories in Ready](https://badge.waffle.io/AzureAD/passport-azure-ad.png?label=ready&title=Ready)](https://waffle.io/AzureAD/passport-azure-ad)
 
 [![Throughput Graph](https://graphs.waffle.io/AzureAD/passport-azure-ad/throughput.svg)](https://waffle.io/AzureAD/passport-azure-ad/metrics)
 
-## Installation
+## 4. Installation
 
     $ npm install passport-azure-ad
 
-## Usage
+## 5. Usage
 
-### Configure strategy
+This library contains two strategies, OIDCStrategy and BearerStrategy.
 
-This sample uses the OAuth2Bearer Strategy:
+OIDCStrategy is for web application login purpose using OpenID connect protocol. It works in the following way:
```

Comment:
Please fix up the grammar in this section. You can use Word's grammar checker to help with some errors.